chia/daemon/server.py

```diff
@@ -1097,6 +1104,8 @@ async def start_service(self, request: Dict[str, Any]):
             except (subprocess.SubprocessError, IOError):
                 log.exception(f"problem starting {service_command}")
                 error = "start failed"
+        elif already_running:
+            success = True
```

Comment:
I often like to put the short case first.  Also, in this case, the other condition simplifies to `error is None`.

Reply:
Sure